make.js

```diff
@@ -211,29 +208,23 @@ CLI.serverBuild = function(/** @type {{ node: string; task: string }} */ argv) {
 
     // Ensure we wrap build function after generator's changes to store only files that changes after the build 
     const buildTaskWrapped = util.syncGeneratedFilesWrapper(buildTask, genTaskPath, callGenTaskDuringBuild);
-    const nodeVersion = (argv.node && argv.node == "Node20") ? argv.node : "Default";
     const allTasksNode20 = allTasks.filter((taskName) => {
-        return taskName.endsWith("Node20");
+        return getNodeVersion(taskName) == 20;
     });
     const allTasksDefault = allTasks.filter((taskName) => {
-        return !taskName.endsWith("Node20");
+        return getNodeVersion(taskName) != 20;
     });
 
-    if (nodeVersion == "Node20") {
+    if (allTasksNode20.length > 0) {
+        util.installNode('20');
         ensureTool('node', '--version', `v${node20Version}`);
-        allTasksNode20.forEach(taskName => buildTaskWrapped(taskName, allTasksNode20.length));
-    } else if (nodeVersion == "Default") {
-        if (allTasksNode20) {
-            console.log(
-            "==========================================\n" + 
-            "IMPORTANT NOTE: There are additional tasks that need to be build with a different node configuration. \n" + 
-            "Unfortunately, we cannot switch node versions while running make.js.  Run the following commands to continue: \n" +
-            `nvm use ${node20Version}\n` +
-            `node make.js build ${argv.task ? `--task ${argv.task} ` : ''}--node Node20\n` + 
-            "==========================================\n");
-        }
+        allTasksNode20.forEach(taskName => buildTaskWrapped(taskName, allTasksNode20.length, 20));
+
+    } 
+    if (allTasksDefault.length > 0) {
+        util.installNode('10');
         ensureTool('node', '--version', `v${node10Version}`);
-        allTasksDefault.forEach(taskName => buildTaskWrapped(taskName, allTasksDefault.length));
+        allTasksDefault.forEach(taskName => buildTaskWrapped(taskName, allTasksDefault.length, 10));
```

Comment:
what does ", 10" do?  (please remove if unneeded)

Reply:
It will be used to output loggings in buildTask function:
`    banner(`Building task ${taskName} using Node.js ${nodeVersion}`);`